### PR TITLE
Refs 4540: switch to object_uuid for tasks

### PR DESCRIPTION
**Pull request by jlsherrill**
## Summary

Previously we had a 'repository_uuid' column on the tasks table.  This changes it to an 'object_uuid' column and adds an 'object_type' column to track the type.  This will allow for tracking different object types to the task (such as templates).

this will allow for cancellation of template tasks

Note that this creates the new columns but does not delete the old column.  this will happen in a followup PR to prevent errors during rollout.  This will need to be promoted to production before the followup PR is merged.

## Testing steps

automation passing 
admin_tasks still works

### CODE REVIEW

**Comment by jlsherrill** [comment]
https://issues.redhat.com/browse/HMS-4313

**Comment by jlsherrill** [comment]
⚠️ This task currently requires qe-approval, but this PR does not fully resolve the task.  Please contact QE to determine appropriate testing required.

**Comment by swadeley** [comment]
Hi @jlsherrill , I don't see the Admin tab. 
I deployed backend=pr-774-7975280 together with  frontend=pr-318-518a3b3 .

**Comment by jlsherrill** [comment]
https://issues.redhat.com/browse/HMS-4540

File: pkg/config/value_constraints.go [hunk 1]
```diff
@@ -116,3 +116,6 @@ func ValidArchLabel(label string) bool {
 	}
 	return false
 }
+
+const ObjectTypeRepository = "repository_config"
+const ObjectTypeTemplate = "template"
```
Comment (rverdile) [inline pkg/config/value_constraints.go:121]: I'd move these to /config/tasks.go

File: pkg/handler/repositories.go [hunk 2]
```diff
@@ -684,10 +687,11 @@ func (rh *RepositoryHandler) enqueueAddUploadsEvent(c echo.Context, response api
...
 			Uploads:              req.Uploads,
 		},
-		OrgId:          orgID,
-		AccountId:      response.AccountID,
-		RepositoryUUID: &response.RepositoryUUID,
-		RequestID:      c.Response().Header().Get(config.HeaderRequestId),
+		OrgId:      orgID,
+		AccountId:  response.AccountID,
+		ObjectUUID: &response.RepositoryUUID,
+		ObjectType: utils.Ptr(config.ObjectTypeRepository),
```
Comment (rverdile) [inline pkg/handler/repositories.go:693]: could also add the templateUUID and type to the templates.go handler events, like you're doing here

Comment (jlsherrill) [reply pkg/handler/repositories.go:693]: I actually did this as part of https://github.com/content-services/content-sources-backend/pull/777/files   the goal here was just to maintain the status quo, but let me know if you want me to do it as part of this PR

Comment (rverdile) [reply pkg/handler/repositories.go:693]: oh, I missed it there. That's fine then, as long as it's included it somewhere.

**Comment by jlsherrill** [comment]
updated!

**Review by rverdile — APPROVED** [review]
looks good!

**Comment by jlsherrill** [comment]
[test]

**Pull request merged**